src/main/java/com/johan/blignaut/entelect/vitalityincubator/task1/HelloWorldController.java

```diff
@@ -0,0 +1,24 @@
+package com.johan.blignaut.entelect.vitalityincubator.task1;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloWorldController {
+
+    @Autowired
+    private HelloWorld helloWorld;
```

Comment:
Please use constuctor injection instead of field injection.
It seems like a trivial change, but constructor injection makes unit testing much easier.